packages/platform-core/src/__tests__/core.test.ts

```diff
@@ -64,8 +68,23 @@ describe('core', () => {
 
   it('should instantiate array', async () => {
     const tx = await startPlugin(platform)
+    const builder = new Builder(tx.getDb())
+    builder.load(model)
+
+    // console.log(tx.getDb().dump())
 
     const result = await tx.find(core.class.Class, {})
     console.log(result)
   })
+
+  it('should merge ids', () => {
+
+    let mm1 : { [key: string]: { [key: string]: any } } = {}
+    mm1['k1'] = ['value11', 'value12']
+    let mm2 : { [key: string]: { [key: string]: any } } = {}
+    mm2['k2'] = ['value21', 'value22']
```

Comment:
values you assign to `mm1` and `mm2` does not match type definitions, maybe you want 

```typescript
mm1['k1'] = { key11: 'value' }`?
```